src/BaseStyles.tsx

```diff
@@ -38,10 +38,33 @@ export type BaseStylesProps = ComponentProps<typeof Base>
 function BaseStyles(props: BaseStylesProps) {
   const {children, color = 'fg.default', fontFamily = 'normal', lineHeight = 'default', ...rest} = props
 
-  const {colorScheme} = useTheme()
+  const {colorScheme, colorMode, dayScheme, nightScheme} = useTheme()
+
+  /**
+   * We need to map valid primer/react color modes onto valid color modes for primer/primitives
+   * valid color modes for primer/primitives: auto | light | dark
+   * valid color modes for primer/primer: auto | day | night | light | dark
+   */
+  type colorModesForPrimitives = 'auto' | 'light' | 'dark'
+  const primerColorModeToPrimitiveColorMode: {[key in ColorModeWithAuto]: colorModesForPrimitives} = {
```

Comment:
Im a little confused about "day" and "night". Can we standardize on just using light/dark/auto? https://github.com/primer/primitives/blob/main/docs/storybook/.storybook/preview.js#L35

Reply:
Would love to 😭 but we're stuck with an extra day and night for backward compatibility. Created [an issue](https://github.com/primer/react/issues/3246) for next major release